base-ci-resource.yaml

```diff
@@ -35,7 +35,7 @@ stages:
 
 #Base components for setting up a deploy configuration.
 .deploy:
-  image: briangweber/docker-node:carbon
+  image: briangweber/docker-node:10
```

Comment:
It would be great if there were a node-12 version of this. Maybe also if it wasn't under Brian's personal Docker Hub space, though it looks like Kevin was able to make a new version of this.

Reply:
Yes... It would be great if someone made it open source/internal open source, or we used an existing one open source image. 

Alternatively, all of these tasks other than publish shouldn't need docker, could be swapped to raw node. Could use an existing open-source docker one if we didn't need both docker and node in the same file for publish